src/test/functional/code/despl.ts

```diff
@@ -10,4 +10,14 @@ AND  R8 R5 R6
 NOR  R9 R5 R6
 XOR  R10 R5 R6`;
 
+export const vliwCodeInput = `
+2  0 0 0 0  1 0 1 0
+0
+2  2 0 0 0  3 0 1 0
+2  4 0 0 0  5 0 1 0
+0
+2  5 0 0 0  6 0 1 0
+2  7 0 0 0  8 0 1 0
+1  9 0 0 0`.trim();
+
 export const resultContent = [3, 2, 12, 0, 11, 6, 15, 2, -16, 13];
```

Comment:
Add a newline at the end of this file!

Reply:
Ups, It always happens to me that I forgot at least one file for Biome